frontend/src/modules/application/actions.js

```diff
@@ -1,10 +1,14 @@
-import { UPDATE_FIELD } from "./constants"
+import { 
+    UPDATE_FIELD, 
+    FETCH_POSITIONS_SUCCESS
+} from "./constants"
 import { error, success  } from "react-notification-system-redux"
 import { errorProps } from "../notifications/constants"
 
 // an action generator function that returns an action object
 export const updateField = data => ({ type: UPDATE_FIELD, data })
 
+// action used by applicant form view
 export const createNewApplication = payload => async dispatch => {
     const res = await fetch("/api/v1/applicants", {
         method: "POST",
```

Comment:
We need to add the credential sending to this request. Once Shibboleth is activated, all requests sent without credentials will fail.